Extension/WindowsForms-Extension/Template-Studio.md

```diff
@@ -117,8 +117,8 @@ Here's a simple explanation:
      - The constructor of PageService takes an IServiceProvider to access the registered services. This allows the service to resolve and instantiate the required view models and views.
 
      II. Registering Pages:
-     - **Configure<DataGridPage>():** Registers DataGridPage with the service provider.
-     - **Configure<MainPage>():** Registers MainPage with the service provider.
+     - **Configure `<DataGridPage>()`:** Registers DataGridPage with the service provider.
```

Comment:
For to resolve below issue Modified this changes in MD file:
![image](https://github.com/user-attachments/assets/80a14eb6-3bc2-4739-8156-1d9157e3bce7)